libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -211,9 +254,16 @@ ptrdiff_t framing::handle(uint8_t opcode, byte_span payload,
                           size_t frame_size) {
   // opcodes are checked for validity when decoding the header
   switch (opcode) {
-    case detail::rfc6455::connection_close:
+    case detail::rfc6455::connection_close: {
+      auto result = decode_closing_payload(payload);
+      if (!result) {
+        abort(result.error());
+        shutdown(result.error());
```

Comment:
Why isn't this calling `abort_and_shutdown`?

Reply:
My mistake.